Mac-App/Mac-App/Transformers/DetailsViewTransformer.swift

```diff
@@ -52,4 +52,3 @@ extension DetailsViewTransformer {
         case failure(_ failure: String)
     }
 }
-
```

Comment:
Every time the tests are run for all targets SwiftFormat is run to auto correct code where possible. 
For now it's fine but maybe this process can be improved without being too intrusive while developing.